src/Symfony/Contracts/Service/LocaleAwareInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Contracts\Service;
```

Comment:
should it actually be in the Service namespace here ? It is not specific to a service container AFAICT.

Reply:
This namespace always has a `ResetInterface` which provides a utilitary functionality for services. So I thought `LocaleAwareInterface` (which also provides a functionality in the same vein) should be here. Any ideas?

Reply:
im not sure `Service` is NS is bound to DI/containers per se. I think we can put any service descriptor here..